web_external/ImagesGallery/Facets/imagesFacet.jade

```diff
@@ -1,6 +1,10 @@
-.attributeSection
-  .header
-    i.toggle.icon-down-open
-    .title= title
-.isic-images-facet-content
+.isic-images-facet-heading(
+  data-toggle='collapse'
+  data-target='##{facetContentId}'
+  )
+  i.isic-images-facet-indicator.icon-down-open
+  span.isic-images-facet-title= title
+.isic-images-facet-content.collapse.in(
+  id='#{facetContentId}'
```

Comment:
PUG doesn't allow this syntax anymore. It will need to be just `id=facetContentId`. Let's change it now to make migration easier.